### DISCUSSION TITLE
Option to add parameters to SQL query. Возможность добавить параметризовать запрос.

### DISCUSSION THREAD

**Discussion by IgorZhurbenko** [ClickHouse/ClickHouse#33227] [Q&A] [answered]
What I struggle to find is a way to protect myself from SQL injection when I compose my query to ClickHouse.
Whereas everywhere else I'd write something like [row].[column] like @param and pass the param value as argument to client method invocation, I don't see that option in Click. What should I do? I write on .Net.

Не могу найти способа передать строку параметром в запрос в Клик, чтобы защититься от SQL инъекций. Хотелось бы написать что-то типа [row].[column] like @param и передать значение параметру в объекте, но никак не получается. Я пишу под .Net.

**Comment by alexey-milovidov** [comment, marked as answer]
https://clickhouse.com/docs/en/interfaces/http/#cli-queries-with-parameters
